pulp_2_tests/tests/rpm/api_v2/test_copy.py

```diff
@@ -230,3 +232,198 @@ def test_all(self):
             1
         )
         self.assertGreater(len(yum_meta_data_element), 0)
+
+
+class CopyConservativeTestCase(unittest.TestCase):
+    """Test ``recursive`` and ``recursive_conservative`` flags during copy.
+
+    RPM packages used in this test case::
+
+        chimpanzee
+        ├── squirrel
+        │   ├── camel
+        │   └── fox
+        └── walrus
+
+    chimpanzee has as dependencies: squirrel and walrus RPM packages.
+    squirrel has as dependencies: camel and fox RPM packages.
```

Comment:
```suggestion
    squirrel has dependencies: camel and fox RPM packages.
```
Reading `has as` just struck me weird.